packages/core-mobile/app/store/settings/appearance/slice.ts

```diff
@@ -10,6 +11,9 @@ export const appearanceSlice = createSlice({
   reducers: {
     setSelectedAppearance: (state, action: PayloadAction<Appearance>) => {
       state.selected = action.payload
+    },
+    setSelectedColorScheme: (state, action: PayloadAction<ColorSchemeName>) => {
+      state.colorScheme = action.payload
     }
```

Comment:
can you add a comment clarifying what selected color scheme is used for and how it is different from appearance?